src/types/user.ts

```diff
@@ -14,6 +14,8 @@ import {
 import { IResponseErrorValidation } from "@pagopa/ts-commons/lib/responses";
 import { DOMParser } from "xmldom";
 
+import { Either } from "fp-ts/lib/Either";
```

Comment:
Either is imported on line 7 as E.Either.

Reply:
fixed. thnak you!